content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb

```diff
@@ -18,6 +19,28 @@ def self.log_and_exit(msg, exit_code, handle = $evm)
               handle.log('info', "Script ending #{msg} code : #{exit_code}")
               exit(exit_code)
             end
+
+            # If you want to create a notification and log a message without specifying a handle using the global $evm
+            #    ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_notify(:level, msg, subject)
+            #
+            # If you want to create a notification and log a message using a specific handle
+            #    ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_notify(:level, msg, subject, handle)
+            #
+            def self.log_and_notify(level, message, subject, handle = $evm)
```

Comment:
@billfitzgerald0120 We need to decide if the level passed into this method is a notify level or a log_level.

When this function is called will the user pass in level to be one of these ("info","error","warn","debug") or would it pass in the notify level ("success","error","warning","info") We have to pick one.

I was thinking it would be the notify level and then we would map from the notify_level to the log_level
so when we call 
``` ruby
def self.log_and_notify(notify_level, message, subject, handle = $evm)
  raise "Invalid notify level #{notify_level}" unless NOTIFY_LEVEL_TO_LOG_LEVEL.keys.include?(notify_level.downcase)
  handle.create_notification(:level => notify_level.downcase, :message => message, :subject => subject)
  handle.log(NOTIFY_LOG_TO_LOG_LEVEL(notify_level.downcase), message)
end
```